libipl/libipl.C

```diff
@@ -359,3 +363,29 @@ void ipl_error_callback(const ipl_error_info& error)
 		return;
 	g_ipl_error_callback_fn(error);
 }
+
+void ipl_plat_clock_error_handler(
```

Comment:
This should moved under p10 folder , common.C

Reply:
done